src/main/twirl/com/lightbend/lagom/docs/faq.scala.html

```diff
@@ -0,0 +1,57 @@
+@import com.lightbend.lagom.docs.LagomContext
+
+@(implicit context: LagomContext)
+
+@page("FAQ") {
+  <div>
+    <h1>Frequently Asked Questions</h1>
+    <blockquote>
+    Have a question about Lagom? We've sorted out a sample of frequently asked questions and provided answers below. If your question is not covered here, try these resources:
+    <ul>
+      <li><a href="//gitter.im/lagom/lagom">Lagom Gitter channel</a>
+      <li><a href="//groups.google.com/forum/#!forum/lagom-framework">lagom-framework mailing list</a> </li>
+      <li><a href="//stackoverflow.com/questions/tagged/lagom">Stackoverflow Lagom questions.</a></li>
+    </blockquote>
+
+
+      <h3>Is Lagom open source or commercial?</h3>  Lagom is
+      completely open source and has a vibrant and growing community. However,
+      you do have the option to purchase a <a href="https://www.lightbend.com/platform/subscription">Lightbend subscription</a> that
+      provides technical support as well as license to use Enterprise Suite,
+      which includes a range of powerful Application Management, Intelligent
+      Monitoring, Enterprise Integration and Advanced Tooling.     </p>
+      <h3>Does Lagom require me to use
+    Cassandra for persistence?</h2> 
+    <p> No. However, Cassandra
+    provides scalability and replication, which is why Lagom uses it by default
+    for storing event journals. We recommend using projections to satisfy the
+    read-side of your application.  For read-side, Lagom supports MySQL,
+    PostreSQL, Oracle, and H2 through JDBC or JPA  see
+    <a href="https://www.lagomframework.com/documentation/1.3.x/java/PersistentEntity.html#Choosing-a-database">Choosing a database</a>
+    and
+    <a href="https://www.lagomframework.com/documentation/1.3.x/java/ReadSideRDBMS.html#Choosing-between-JDBC-and-JPA">Choosing between JDBC and JPA</a>
+    for more information. </p>      
+
+        <h3>In the Java API, why are event interfaces present in
+        both API and impl classes with only slight differences between them?
+      </h3> <p>This is an example of an anti-corruption layer (ACL). In Lagom, we use
```

Comment:
I don't want to use the abbreviation "ACL" in this context, because we also use it elsewhere in Lagom to stand for "access control list", and I think the duplication could be very confusing.